### PR TITLE
Moat loader

### PR DESCRIPTION
**Pull request by giannif**
integrate moat

### CODE REVIEW

File: packages/react-components/src/components/gif.tsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import { IGif, IUser } from '@giphy/js-types'
 import { checkIfWebP, getAltText, getBestRenditionUrl, getGifHeight, Logger } from '@giphy/js-util'
+import moat from '@giphy/moat-loader'
 import { css, cx } from 'emotion'
 import React, { ReactType, SyntheticEvent, useEffect, useRef, useState } from 'react'
+import constructMoatData from '../util/construct-moat-data'
 import * as pingback from '../util/pingback'
 import AdPill from './ad-pill'
 
+const moatLoader = moat.loadMoatTag('giphydisplay879451385633')
```
Comment (giannif) [inline packages/react-components/src/components/gif.tsx]: @cosmocochrane1 is this right?

Comment (giannif) [reply packages/react-components/src/components/gif.tsx]: @cosmocochrane1 it's not. did we update moat loader to use our bucket script?

Comment (cosmocochrane1) [reply packages/react-components/src/components/gif.tsx]: we're waiting on moat to send us the new dynamic loader that pulls from our moattag

Comment (cosmocochrane1) [reply packages/react-components/src/components/gif.tsx]: for now this is what they said to use

**Pull request merged**